lib/cassandra_migrations/cassandra/keyspace_operations.rb

```diff
@@ -7,16 +7,18 @@ module KeyspaceOperations
 
       def create_keyspace!(env)
         config = Config.configurations[env]
+        validate_config(config)
+
+        execute(
+          "CREATE KEYSPACE #{config.keyspace} \
+           WITH replication = { \
+             'class':'#{config.replication['class']}', \
+             'replication_factor': #{config.replication['replication_factor']} \
+           }"
+        )
```

Comment:
Note that this execute call was pulled outside the error block.  My thoughts:

* Any failure in executing the keyspace is exceptional, since we validate the configuration and expect it to succeed.
* The previous flow hid any exception that occurred when creating the keyspace, since the rescue block tries to drop the keyspace (which will fail with `UnexistingKeyspaceError` if the creation failed).